client/dashboard/utils/site-status.ts

```diff
@@ -4,11 +4,11 @@ import type { Site } from '../data/types';
 export const STATUS_LABELS = {
 	public: __( 'Public' ),
 	private: __( 'Private' ),
-	coming_soon: __( 'Coming Soon' ),
+	coming_soon: __( 'Coming soon' ),
```

Comment:
I've changed these status labels to sentence case to match Figma. But for some reason `__( 'Coming soon' ) === 'Coming Soon'`!! I can't figure out what's going on. Any ideas @Automattic/lego?

Reply:
Oh maybe it's this

https://translate.wordpress.com/projects/wpcom/en-gb/default?filters%5Bterm%5D=Coming+soon&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filters%5Bview%5D=&filter=Filter&sort%5Bby%5D=priority&sort%5Bhow%5D=desc

It looks like "Coming Soon" has been provided as a translation for "Coming soon" in `en-gb` locale 🤨

Reply:
I've corrected the `en-gb` translation.

The only place this translation is used outside the v2 dashboard is here:

![CleanShot 2025-06-18 at 12 05 01@2x](https://github.com/user-attachments/assets/63052209-2cfa-4cd7-ab50-408ab7fa5054)

And it's ok for this to switch to sentence caps too ("All sites" is already sentence caps)